pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
The only reason for a label is to work with the scheduler's storage topologyKey mechanism, which operates on labels.